app/views/projects/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Project</h1>
+
+<p>Project Name: <%= @project.name %> </p>
+<p>Project Material: <%= @project.material %> </p>
+<p>Challege Theme: <%= @theme %> </p>
+<p>Number of Contestants: <%= @project.num_contestants %></p>
+<p>Average Contestant Experience: <%= @project.ave_exp %></p>
+
+<%= form_with url: "/projects/#{@project.id}", method: :patch, local: true do |form| %>
```

Comment:
In conjunction with my comments in your commented out controller action: a more RESTful path  here could be a POST (bc we're creating a joins record) to something like "/projects/:project_id/contestants" -- this indicates that you're dealing with both of these resources and this data operation is associating two of these records together.